drivers/espi/espi_mchp_xec.c

```diff
@@ -397,6 +406,72 @@ static int espi_xec_receive_vwire(struct device *dev,
 	return 0;
 }
 
+static int espi_xec_send_oob(struct device *dev, struct espi_oob_packet pckt)
+{
+	int ret;
+	struct espi_xec_data *data = (struct espi_xec_data *)(dev->driver_data);
+	u8_t err_mask = MCHP_ESPI_OOB_TX_STS_IBERR |
+			MCHP_ESPI_OOB_TX_STS_OVRUN |
+			MCHP_ESPI_OOB_TX_STS_BADREQ;
+
+	LOG_DBG("%s\n", __func__);
+
+	if (!(ESPI_OOB_REGS->TX_STS & MCHP_ESPI_OOB_TX_STS_CHEN)) {
+		LOG_WRN("OOB channel is disabled\n");
+		return -EIO;
+	}
+
+	if (ESPI_OOB_REGS->TX_STS & MCHP_ESPI_OOB_TX_STS_BUSY) {
+		LOG_WRN("OOB channel is busy\n");
+		return -EBUSY;
+	}
+
+	memcpy(&slave_tx_mem, pckt.buf, pckt.len);
```

Comment:
I think we should make sure that the packet can actually fit inside `slave_tx_mem`. Also remove `&`.